tests/Metadata/MetadataProviderTest.php

```
@@ -449,11 +596,199 @@ public function testWarnsAboutDoubleWrappingWhenClientAndBothCacheAndLoggerProvi
+
+        // Initialize ApiClient with BOTH httpClient AND cache
+        ApiClient::getInstance([
+            'apiUrl'     => self::API_URL,
+            'httpClient' => $httpClient,
+            'cache'      => $cache
+        ]);
+
+        restore_error_handler();
+
+        // Set up error handler to catch MetadataProvider warnings
+        set_error_handler(function (int $_errno, string $_errstr) use (&$warningIssued): bool {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$_errno'. (undefined)

(UnusedFormalParameter)